doc/sphinx/changelog.rst

```diff
@@ -10,57 +18,57 @@ New
 - Initial release of the pagmo/pygmo C++ software-development kit (SDK). The purpose of the SDK is to make the process
   of writing C++ extensions for pagmo/pygmo as easy as possible. The SDK is a beta-quality feature at this time,
   and it is lightly documented - no tutorials are available yet. Please come to the gitter channel and ask there if you are interested
-  in it (`#110 <https://github.com/esa/pagmo2/pull/110>`_).
-
-- Improve support for integer and mixed integer optimization (`#115 <https://github.com/esa/pagmo2/pull/115>`_).
+  in it (`#110 <https://github.com/esa/pagmo2/pull/110>`__).
```

Comment:
Can you add to this PR something like following text: "The problem integer part is no longer available as argument when constructing algorithms such as pagmo::sga and pagmo::nsga2, it is must instead be specified implementing a method get_nix on the UDP". 

Since this breaks previous code it may be good to have it logged.